### PR TITLE
feat(488): transaction log

### PR DESCRIPTION
**Pull request by dexturr**
# Related issues 🔗

https://github.com/vegaprotocol/vegawallet-desktop/issues/488

# Description ℹ️

Adds Individual transaction page for the transaction log, merely the mocked UI for now. Wiring coming in the next PR :+1:

# Demo 📺

https://user-images.githubusercontent.com/26136207/229475979-5229dc5a-8119-4633-aa93-437e635a9057.mov

# Technical 👨‍🔧

Tests are commented out because of a dependecy issue with UI toolkit that needs to be resolved and will be done as a follow up to this PR.

### CODE REVIEW

File: libs/wallet-ui/src/components/code-window/code-window.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { render, screen } from '@testing-library/react'
+import { CodeWindow } from './code-window'
+
+jest.mock('../copy-with-tooltip', () => ({
+  CopyWithTooltip: () => <div data-testid="copy-button" />,
+}))
+
+const renderComponent = () => {
+  return render(
+    <CodeWindow
```
Comment (edd) [inline libs/wallet-ui/src/components/code-window/code-window.spec.tsx:10]: **praise (non-blocking)**: I think the raw JSON view is more useful here than adopting / adapting components from the Block Explorer - the protos are generally pretty succinct.

File: libs/wallet-ui/src/components/collapsible-panel/collapsible-pabel.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import { fireEvent, render, screen } from '@testing-library/react'
```
Comment (edd) [inline libs/wallet-ui/src/components/collapsible-panel/collapsible-pabel.spec.tsx:1]: **chore (non-blocking)**: This should be `collapsible-panel.spec.tx` not `-pabel-`

File: libs/wallet-ui/src/components/navbar/index.tsx [hunk 3]
```diff
@@ -1,9 +1,10 @@
 import type { ReactNode } from 'react'
-import { Settings } from '../icons/settings'
+// import { Settings } from '../icons/settings'
```
Comment (edd) [inline libs/wallet-ui/src/components/navbar/index.tsx:2]: **nitpick (non-blocking)**: Normally 🤨 , but it was explained in chat that this PR is WIP, but needs to be merged early due to some dependency complications. So 🤨 ✅.

File: libs/wallet-ui/src/components/collapsible-panel/collapsible-pabel.spec.tsx (hunk 2 above)
Comment (dexturr) [reply libs/wallet-ui/src/components/collapsible-panel/collapsible-pabel.spec.tsx:1]: I've got an immediate follow up to this PR I am going to address this in there

**Pull request merged**